bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConfiguration.java

```diff
@@ -203,6 +199,10 @@ public Builder toBuilder() {
   }
 
   public boolean equals(Object obj) {
-    return EqualsBuilder.reflectionEquals(this, obj);
+    if (obj == null || obj.getClass() != this.getClass()) {
```

Comment:
If it's not the same class (i.e. if objs or this is a different subclass), then the objects are not equals.